relayer/cmd/generate_beacon_data.go

```diff
@@ -172,11 +173,19 @@ func generateBeaconData(cmd *cobra.Command, _ []string) error {
 			return fmt.Errorf("get initial sync: %w", err)
 		}
 		initialSync := initialSyncScale.ToJSON()
+		err = writeJSONToFile(initialSync, fmt.Sprintf("initial-checkpoint.%s.json", activeSpec.ToString()))
 		initialSyncHeaderSlot := initialSync.Header.Slot
+		initialSyncPeriod := s.ComputeSyncPeriodAtSlot(initialSyncHeaderSlot)
 		log.Info("created initial sync file")
 
 		log.Info("downloading beacon state, this can take a few minutes...")
-		syncCommitteePeriod := s.ComputeSyncPeriodAtSlot(initialSyncHeaderSlot)
+		// wait for 5 blocks
+		time.Sleep(6 * time.Second * 5)
```

Comment:
Is the block time 6s in our E2E stack? Curious if we configured this explicitly somewhere

You should probably make it wait some more time in case the 5th block hasn't been produced yet.

Reply:
Yes, it's 6s for minimal spec and we did not hack it for this test. Waiting for 5 blocks is just a decent value to make sure `FinalizedUpdate` change and `SyncPeriod` does not change.  This is requied by recently added checks in our light client.
There is chance at some time we run command fail to get data meet requirements,  in that case we just fail with error as following.
https://github.com/Snowfork/snowbridge/blob/3accd341008bc0743fc972b8b10663ecc032dc09/relayer/cmd/generate_beacon_data.go#L214-L219